### DISCUSSION TITLE
Potential bug on UI: duplicate tables

### DISCUSSION THREAD

**Discussion by ntkris** [supabase/supabase#12113] [General]
Keep seeing duplicate tables on the UI. Might be a bug?

<img width="235" alt="image" src="https://user-images.githubusercontent.com/32306480/216198159-e453f794-fbb7-4725-a2cd-eb7d8fa0dc48.png">

**Comment by Hallidayo** [comment]
Hi @ntkris - Just checking if this issue is still happening?
